### PR TITLE
Adding Titles of Modules and sw-routing docs

### PR DESCRIPTION
**Pull request by gauntface**
R: @jeffposnick @jpmedley 

Adding sw-routing docs.

### CODE REVIEW

File: packages/sw-background-sync-queue/src/index.js [hunk 1]
```diff
@@ -12,7 +12,10 @@
  */
 
 /**
- * sw-background-sync-queue Module
+ * # sw-background-sync-queue
+ *
+ * A service worker implementation of the a queue which is triggered by
```
Comment (jpmedley) [inline packages/sw-background-sync-queue/src/index.js]: 'a queue' or 'the queue'?

File: packages/sw-routing/src/lib/express-route.js [hunk 2]
```diff
@@ -50,16 +47,22 @@ import pathToRegExp from 'path-to-regexp';
...
+   * @param {Object} input
+   * @param {string} input.path The path to use for routing.
    *        If the path contains [named parameters](https://github.com/pillarjs/path-to-regexp#named-parameters),
    *        then an Object mapping parameter names to the corresponding value
    *        will be passed to the handler via `params`.
-   * @param {Object} handler - An Object with a `handle` method. That method
-   *        will be used to respond to matching requests.
-   * @param {string} [method] Only match requests that use this
+   * @param {Object} input.handler - An Object with a `handle` method. That
+   *        method will be used to respond to matching requests.
```
Comment (jpmedley) [inline packages/sw-routing/src/lib/express-route.js]: An Object with a `handle` method that will be used to respond to matching requests.

Combine the pieces into one.

File: packages/sw-background-sync-queue/src/index.js (hunk 1 above)
Comment (gauntface) [inline packages/sw-background-sync-queue/src/index.js]: This is copied straight out of package.json - not my words. @prateekbh ?

File: packages/sw-routing/src/lib/express-route.js (hunk 2 above)
Comment (gauntface) [inline packages/sw-routing/src/lib/express-route.js]: Done.

File: packages/sw-background-sync-queue/src/index.js (hunk 1 above)
Comment (gauntface) [inline packages/sw-background-sync-queue/src/index.js]: Reading this description I'm not convinced it's the best way to describe the module.

Would something along the lines of:

"Queues failed requests and uses the Background Sync API to replay those requests at a later time when the network state has changed."

Comment (jpmedley) [inline packages/sw-background-sync-queue/src/index.js]: That sounds good.

**Comment by gauntface** [comment]
@jpmedley that should be the last change in.

**Pull request merged**